cassandane/tiny-tests/calendar/account-get-capabilities

```diff
@@ -0,0 +1,52 @@
+#!perl
```

Comment:
Historically, we have a massive copyright comment blob at the top of every file.  I'd love to be rid of them...

Reply:
I can put them back if we want (or need) to, but I'm hoping we can avoid it. :)

Reply:
I'm not comfortable making copyright decisions, so I just meticulously copy what was already being done, rather than risk becoming responsible for a bad choice. 😅 

Interestingly, this is Cassandane, which is variously Copyright Fastmail/Opera, _not_ CMU, so the calculus is different from what it would be for Cyrus itself.

I do not like having copyright statements at the top of every file, but I will defer to whatever decision someone else makes.  I bring up the issue to observe that there's a decision point here, so it can be made consciously rather than accidentally.